workflows/prognostic_c48_run/runtime/radiation_wrapper.py

```diff
@@ -0,0 +1,484 @@
+import dataclasses
+from typing import Optional, Mapping, Any, Literal, Tuple, Sequence, Hashable
+import cftime
+from datetime import timedelta
+import numpy as np
+import xarray as xr
+from runtime.steppers.machine_learning import (
+    MachineLearningConfig,
+    open_model,
+    MultiModelAdapter,
+    predict,
+)
+from runtime.types import State
+from runtime.derived_state import DerivedFV3State
+from radiation import RadiationDriver, getdata, NGPTSW, NGPTLW
+from vcm.calc.thermo.vertically_dependent import (
+    pressure_at_interface,
+    pressure_at_midpoint_log,
+)
+from vcm.calc.thermo.constants import _SPECIFIC_HEAT_CONST_PRESSURE, _RDGAS
+
+P_REF = 1.0e5
+MINUTES_PER_HOUR = 60.0
+SECONDS_PER_MINUTE = 60.0
+
+DEFAULT_RAD_CONFIG = {
+    "imp_physics": 11,
+    "iemsflg": 1,  # surface emissivity control flag
+    "ioznflg": 7,  # ozone data source control flag
+    "ictmflg": 1,  # data ic time/date control flag
+    "isolar": 2,  # solar constant control flag
+    "ico2flg": 2,  # co2 data source control flag
+    "iaerflg": 111,  # volcanic aerosols
+    "ialbflg": 1,  # surface albedo control flag
+    "icldflg": 1,
+    "ivflip": 1,  # vertical index direction control flag
+    "iovrsw": 1,  # cloud overlapping control flag for sw
+    "iovrlw": 1,  # cloud overlapping control flag for lw
+    "isubcsw": 2,  # sub-column cloud approx flag in sw radiation
+    "isubclw": 2,  # sub-column cloud approx flag in lw radiation
+    "lcrick": False,  # control flag for eliminating CRICK
+    "lcnorm": False,  # control flag for in-cld condensate
+    "lnoprec": False,  # precip effect on radiation flag (ferrier microphysics)
+    "iswcliq": 1,  # optical property for liquid clouds for sw
+    "lsswr": True,
+    "lslwr": True,
+    "nfxr": 45,
+    "ncld": 5,
+    "ncnd": 5,
+    "lgfdlmprad": False,
+    "uni_cld": False,
+    "effr_in": False,
+    "indcld": -1,
+    "num_p3d": 1,
+    "npdf3d": 0,
+    "ncnvcld3d": 0,
+    "lmfdeep2": True,
+    "lmfshal": True,
+    "sup": 1.0,
+    "kdt": 1,
+    "do_sfcperts": False,
+    "pertalb": [[-999.0], [-999.0], [-999.0], [-999.0], [-999.0]],
+    "do_only_clearsky_rad": False,
+    "swhtr": True,
+    "solcon": 1320.8872136343873,
+    "lprnt": False,
+    "lwhtr": True,
+    "lssav": True,
+}
+
+
+@dataclasses.dataclass
+class RadiationWrapperConfig:
+    """"""
+
+    kind: Literal["python"]
+    input_model: Optional[MachineLearningConfig] = None
+
+
+class RadiationWrapper:
```

Comment:
Can you make this a Stepper and hide some of the idiosyncratic and redundantly named methods e.g. `rad_init`, `rad_update` and `rad_compute`? Do you really need separate concepts for `compute` and `update`? That is the main departure from the Stepper API.

Reply:
Probably not. I think they could be combined into the stepper `__call__` as you suggest.